src/quantuminspire/api.py

```diff
@@ -472,7 +473,7 @@ def _create_job(self, name: str, asset: Dict[str, Any], project: Dict[str, Any],
             'full_state_projection': full_state_projection,
             'user_data': user_data
         }
-        if not full_state_projection and self.enable_fsp_warning:
+        if not full_state_projection and self.enable_fsp_warning and not backend_type["is_hardware_backend"]:
```

Comment:
```suggestion
        if not full_state_projection and self.enable_fsp_warning and not backend_type.get("is_hardware_backend", False):
```

Reply:
Looks good. It's safer the way you propose.